static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -350,6 +351,14 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
         >
           {t('Tell us where your source code is')}
         </FixMappingButton>
+        {frame.sourceLink && (
+          <React.Fragment>
+            <IconFile />
+            <OpenInLink onClick={onOpenLink} href={frame.sourceLink} openInNewTab>
+              {t('or open in source')}
+            </OpenInLink>
+          </React.Fragment>
```

Comment:
temp example of using sourceLink when stacktrace linking is enabled but no provider integrations linked yet

Reply:
Do we want sourceLink to display always if there is no provider integration (ie. don't bother displaying "Tell us where your code is" as long as there is a source link available)? Or should we still give users the option to link source code (not sure if this is actually fully functional for C#)